RecoTracker/ConversionSeedGenerators/python/ConversionStep2_cff.py

```diff
@@ -222,8 +222,7 @@
 conv2CkfTrajectoryBuilder = RecoTracker.CkfPattern.GroupedCkfTrajectoryBuilder_cfi.GroupedCkfTrajectoryBuilder.clone(
     trajectoryFilter = dict(refToPSet_ = 'conv2CkfTrajectoryFilter'),
     minNrOfHitsForRebuild = 3,
-    clustersToSkip        = cms.InputTag('conv2Clusters'),
```

Comment:
Here, too, the parameter is removed b/c `GroupedCkfTrajectoryBuilder` does not have/use a parameter named `clustersToSkip`.